### DISCUSSION TITLE
도전과제, 컬렉션 사용되는 것 추리기

### DISCUSSION THREAD

**Discussion by wlrnfyd0329** [SeoulTechTCPGame/HumanSurvival#121] [Q&A]
도전과제와 컬렉션 부분에서 저희가 무기와 캐릭터, 아르카나를 간추리는 부분에서 사용하지 않는 부분이 있는데 무엇이 사용되고 필요한지 알려주시면 제가 반영해서 MainCollection, MainAchievements를 수정하도록 하겠습니다.
예를 들어 사용하는 것이 사용하지 않는 것보다 적으니까 컬렉션, 도전과제에서 몇번, 몇번 사용되는지 번호로 해주시면 조치하도록 하겠습니다.

**Comment by hgsanguk** [comment]
위키의 _[잠금 해제 및 컬렉션 리스트](https://github.com/SUST22Unity/HumanSurvival/wiki/%EC%9E%A0%EA%B8%88-%ED%95%B4%EC%A0%9C-%EB%B0%8F-%EC%BB%AC%EB%A0%89%EC%85%98-%EB%A6%AC%EC%8A%A4%ED%8A%B8)_ 문서에 추려서 리스트를 작성했습니다.
